doc/user/data/self_managed/latest_versions.yml

```diff
@@ -2,6 +2,6 @@ terraform_gcp_version: v0.4.2
 terraform_azure_version: v0.4.2
 terraform_aws_version: v0.4.5
 terraform_helm_version: v0.1.14
-operator_helm_chart_version: v25.1.12
-orchestratord_version: v0.144.0
-environmentd_version: v0.130.13
+operator_helm_chart_version: v25.2.0
```

Comment:
So ... I believe Dennis' release script automagically update these when we publish the helm chart.

https://github.com/MaterializeInc/materialize/blob/main/misc/helm-charts/publish.sh